pkgs/by-name/ra/rainbowcrack/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitLab
+, alglib
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "rainbowcrack";
+  version = "1.8-0kali1";
+
+  src = fetchFromGitLab {
+    group = "kalilinux";
+    owner = "packages";
+    repo = "rainbowcrack";
+    rev = "kali/${version}";
+    hash = "sha256-y6jdFVQSO80YLLOitPAY8TVMUsi58B6+3bFoncwr+Qo=";
+  };
+
+  buildInputs = [ alglib ];
+
+  installPhase = ''
+    runHook preinstall
```

```suggestion
    runHook preInstall
```

i am pretty bash is case sensetive